examples/tinywl-new/wallpaperimage.cpp

```diff
@@ -0,0 +1,100 @@
+// Copyright (C) 2024 WenHao Peng <pengwenhao@uniontech.com>.
+// SPDX-License-Identifier: Apache-2.0 OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+
+#include "wallpaperprovider.h"
+#include "wallpaperimage.h"
+#include "helper.h"
+
+WallpaperImage::WallpaperImage(QQuickItem *parent)
+    :QQuickImage(parent)
+{
+    auto provider = Helper::instance()->qmlEngine()->wallpaperImageProvider();
+    connect(provider, &WallpaperImageProvider::wallpaperTextureUpdate, this, &WallpaperImage::updateWallpaperTexture);
+}
+
+WallpaperImage::~WallpaperImage()
+{
+
+}
+
+QString WallpaperImage::userId()
+{
+    return m_userId;
+}
+
+void WallpaperImage::setUserId(const QString& name)
+{
+    if (m_userId == name) {
+        m_userId = name;
+        Q_EMIT userIdChanged();
+        updateSource();
+    }
+}
+
+QString WallpaperImage::screenId()
+{
+    return m_screenId;
+}
+
+void WallpaperImage::setScreenId(const QString& screen_id)
```

Comment:
Like as `setUserId`, What's screen id? xdg output name?
`void setOutput(Output *output)` is better.